frontend/appflowy_tauri/src/appflowy_app/components/editor/command/index.ts

```diff
@@ -308,6 +308,24 @@ export const CustomEditor = {
     return CustomEditor.getBlockType(editor) === EditorNodeType.GridBlock;
   },
 
+  isRootBlock: (editor: ReactEditor) => {
```

Comment:
How about this:
```ts
isRootBlock: (editor: ReactEditor) => {
  const [match] = Editor.nodes(editor, {
    match: n => 
      Element.isElement(n) && 
      n.blockId !== undefined && 
      n.type === EditorNodeType.Page,
  });

  return Boolean(match);
},

```
1.the loop exists asap
2.using Boolean

Reply:
In fact, this function is used to determine whether the selection contains the root (that is, title) node. For example, if the selection contains root, the toolbar will not be displayed.
Let me change the name of this function